### PR TITLE
samd21: add GPIO support for port B

### PR DESCRIPTION
**Pull request by aykevl**
I think this is not complete yet. Also, not yet tested very well.

### CODE REVIEW

**Comment by deadprogram** [comment]
That is true, for example it still needs the `getPinCfg()` for the PORTB pins.

**Comment by aykevl** [comment]
Yeah it was mostly hacked together. Will update when I get to it. Or feel free to continue the work.

File: src/machine/machine_atsamd21.go [hunk 1]
```diff
@@ -62,16 +73,48 @@ func (p GPIO) Configure(config GPIOConfig) {
 
 // Get returns the current value of a GPIO pin.
 func (p GPIO) Get() bool {
-	return (sam.PORT.IN0>>p.Pin)&1 > 0
+	if p.Pin < 32 {
+		return (sam.PORT.IN0>>p.Pin)&1 > 0
+	} else {
+		return (sam.PORT.IN1>>p.Pin)&1 > 0
```
Comment (aykevl) [inline src/machine/machine_atsamd21.go]: Here's an issue: missing `- 32`

**Comment by deadprogram** [comment]
Once #195 lands, then this one can as well, and I can probably finish the other pin mappings pretty quickly.

**Comment by deadprogram** [comment]
Let's merge this, and then iterate on it from there. Doing it.

**Pull request merged**